package.json

```
@@ -22,59 +22,56 @@
     "check": "dependency-check . --no-dev --ignore-module patch-package --ignore-module websocket-reconnector",
     "coverage": "nyc --lines 95 --functions 95 --branches 95 --reporter=lcov --reporter=text npm test",
     "lint": "eslint --cache .",
-    "precommit": "npm run lint",
-    "prepare": "patch-package",
-    "prepublishOnly": "npm run precommit && npm run prepush && tag-matches",
-    "prepush": "npm run check && npm test",
+    "prepare": "patch-package && husky install",
+    "prepublishOnly": "npm run lint && npm run check && npm test && tag-matches",
     "test": "mocha"
   },
   "dependencies": {
-    "axios": "0.18.1",
-    "axios-cookiejar-support": "^0.4.2",
+    "axios": "0.21.1",
+    "axios-cookiejar-support": "^1.0.1",
     "debug": "^4.1.1",
-    "ethereumjs-wallet": "^0.6.3",
+    "ethereumjs-wallet": "1.0.1",
     "lodash": "^4.17.15",
     "merkletreejs": "0.0.25",
     "metronome-contracts": "^2.5.0",
-    "metronome-sdk-status": "^1.2.0",
-    "p-all": "^2.1.0",
-    "p-defer": "^1.0.0",
-    "p-retry": "^3.0.1",
-    "p-timeout": "^2.0.1",
+    "metronome-sdk-status": "^1.2.1",
+    "p-all": "^3.0.0",
+    "p-defer": "3.0.0",
+    "p-retry": "^4.6.0",
+    "p-timeout": "^4.1.0",
     "p-whilst": "^2.0.0",
     "patch-package": "^5.1.1",
     "promise-all-props": "^1.0.1",
     "safe-exchange-rate": "^1.0.0",
     "socket.io-client": "2.1.1",
     "tough-cookie": "^2.5.0",
-    "web3": "1.2.1",
+    "web3": "1.3.6",
     "websocket-reconnector": "^1.1.1"
   },
   "devDependencies": {
-    "chai": "^4.2.0",
+    "chai": "^4.3.4",
     "chai-as-promised": "^7.1.1",
     "check-tag-matches": "^1.0.0",
     "dependency-check": "^4.1.0",
-    "eslint": "^6.4.0",
-    "eslint-config-bloq": "^2.2.0",
-    "eslint-config-standard": "^12.0.0",
-    "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-jsdoc": "^15.9.2",
-    "eslint-plugin-mocha": "^6.1.1",
-    "eslint-plugin-node": "^10.0.0",
-    "eslint-plugin-prefer-arrow": "^1.1.6",
-    "eslint-plugin-promise": "^4.2.1",
+    "eslint": "^7.29.0",
+    "eslint-config-bloq": "^2.4.0",
+    "eslint-config-standard": "^16.0.3",
+    "eslint-plugin-import": "^2.23.4",
+    "eslint-plugin-jsdoc": "^35.4.1",
+    "eslint-plugin-mocha": "^9.0.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prefer-arrow": "^1.2.3",
+    "eslint-plugin-promise": "^5.1.0",
     "eslint-plugin-require-path-exists": "^1.1.9",
-    "eslint-plugin-standard": "^4.0.1",
-    "husky": "^2.3.0",
-    "mocha": "^6.2.0",
-    "nock": "^10.0.6",
-    "npm-check": "^5.8.0",
-    "nyc": "^14.1.1",
+    "husky": "^6.0.0",
+    "mocha": "^9.0.1",
+    "nock": "^13.0.2",
+    "npm-check": "^5.9.2",
+    "nyc": "^15.1.0",
     "proxyquire": "^2.1.3",
-    "randomstring": "^1.1.5"
+    "randomstring": "^1.2.1"
   },
   "engines": {
-    "node": ">=8 <=10"
+    "node": ">=12 <=14"
```

Why limiting it to 12-14? Does this package fail with i.e. Node.js 16?

```suggestion
    "node": ">=12"
```

---

no, but I did not try 😅

---

I removed the restriction https://github.com/autonomoussoftware/metronome-wallet-core/pull/34/commits/4c656ff999c203f63587f2daa68774507db40809